kitura-init.js

```diff
@@ -34,8 +34,13 @@ validateDirectoryName();
 // Make sure directory is empty.
 checkCurrentDirIsEmpty();
 
-// Clone repo contents into current directory.
-cloneProject(initURL, initBranch);
+// If '--clone' specified, then clone project from provided git repository.
+if ((args.includes('--clone'))) {
+  cloneProject(process.argv[3], initBranch);
```

Comment:
This means we'll always look for an `init` branch, which doesn't seem right. Would it be better to do it like this:

```
kitura init --repo ianpartridge/myproject --branch mybranch
```

So we automatically add `github.com` etc. if it's not provided, and let people specify a branch if they want?  If they don't add `--branch` we use `master` by default.

Reply:
Amended it to default to the master branch, and accept a --branch param.